packages/compass-home/src/components/home.tsx

```diff
@@ -219,7 +219,7 @@ function Home({ appName }: { appName: string }): React.ReactElement | null {
     appRegistry.on('data-service-disconnected', onDataServiceDisconnected);
     appRegistry.on('select-database', onSelectDatabase);
     appRegistry.on('select-namespace', onSelectNamespace);
-    appRegistry.on('select-instance', onSelectInstance);
+    appRegistry.on('sidebar-instance-workspace-open-tab', onSelectInstance);
```

Comment:
should we also rename the handler to match the event name?

Reply:
Yeah, totally, somehow missed it, thanks for highlighting this